### ISSUE TITLE
Feature Request: Limit how many Event types

### ISSUE THREAD

**Issue by Lifeischoice** [dayschedule/public-roadmap#36]
Ability to limit how many event types are booked daily. This will allow good management of how many events of the same type are  available for booking on a daily bases. For example, you can have 3 15 minute "coffee chats" and 4 1 hr client sessions per day.

**Comment by vickyrathee** [comment]
@jainshubh0612 Let's plan this

**Comment by Lifeischoice** [comment]
> @Lifeischoice This is yet to be started. Expected date: Sep 2022

@vickyrathee I know you guys are busy and putting out great features but I will really appreciate if you can move this forward as it is necessary and essential for me to manage my business. I am currently on your enterprise plan and need this feature to transition from Calendly. Or can you suggest a work around? Please see what you can do. Thanks

**Comment by vidushisoni** [comment]
I agree. Desperately in need of a feature which permits to add "Maximum meetings per day". Please update us soon with it. Thank you.

**Comment by doshisahil95** [comment]
+1. Need this feature request as well. Thanks!

**Comment by hmd123456** [comment]
+1  I would like to see it as soon as possible 

This is important feature to throttle the incoming bookings.  There should be minimum two controls . 1> number of customers that can  book the same time slot e.g 20 students can book same time slot  on 8th June 11 AM  regardless of how many users are setup in Dayschedule 2> number of students that they can book time slots in the entire day e.g.  only upto 60 students  should be able to book different time slots on the same day i.e 8th June . Thanks
